tools/ci_build/github/android/build_aar_package.py

```diff
@@ -94,6 +95,22 @@
     base_build_command = [sys.executable, BUILD_PY] + build_settings["build_params"] + ["--config=" + build_config]
     header_files_path = ""
 
+    if qnn_android_build:
+        qnn_home = args.qnn_path
+        sdk_file = os.path.join(qnn_home, "sdk.yaml")
+
+        with open(sdk_file) as f:
+            for line in f:
+                if line.strip().startswith("version:"):
+                    # yaml file has simple key: value format with version as key
+                    qnn_sdk_version = line.split(":", 1)[1].strip()
+                    break
+
+        # only use major.minor.patch version for qnn sdk version and truncate the build_id info if any
+        # yaml file typically has version like 2.26.0
+        qnn_sdk_version = ".".join(qnn_sdk_version.split(".")[:3])
```

Comment:
Would be good to add a check that the version line is found. 

We could also capture the info from QC that the QNN package is not really semver somewhere so that info isn't lost.